torchok/models/backbones/__init__.py

```diff
@@ -6,3 +8,7 @@
 import torchok.models.backbones.mobilenetv3
 import torchok.models.backbones.resnet
 import torchok.models.backbones.swin
+
+has_mmcv = importlib.util.find_spec("mmcv")
```

Comment:
Why we need this? I think just import is ok

Reply:
It is for cases when you do not install mmcv. Without it, CI/CD will fail